FirebaseAdmin/FirebaseAdmin/Auth/FirebaseUserManager.cs

```diff
@@ -60,6 +61,30 @@ public static FirebaseUserManager Create(FirebaseApp app)
             return new FirebaseUserManager(args);
         }
 
+        /// <summary>
+        /// Get a user by id.
+        /// </summary>
+        /// <param name="uid">The query user id.</param>
+        /// <param name="cancellationToken">For asynchronous operation.</param>
+        /// <returns>A record of user with the queried id if one exists.</returns>
```

Comment:
s/queried/given

Reply:
Just wanted to draw attention to this comment. I can merge this PR once this is addressed.

Reply:
@hiranya911 is it possible to merge from [this commit](https://github.com/firebase/firebase-admin-dotnet/pull/32/commits/d44212791b4f4891ed998fd65467ef064ef30c4d). It is the last commit that addressed the comments already

Reply:
It didn't change `queried` to `given` as far as I can see. In any case, this is a minor nit.  I will merge this PR tomorrow regardless of that change.